macros/PGbasicmacros.pl

```diff
@@ -2112,7 +2112,7 @@ sub general_math_ev3 {
 
 	my $out;
 	if($displayMode eq "HTML_MathJax") {
-		$out = '<span>'.$in_delim.'</span>';
+		$out = '<span>'.encode_pg_and_html($in_delim).'</span>';
```

Comment:
Although this should work, it is more aggressive than necessary, as it encodes lots of other things that don't need to be.  For example, it encodes the backslashes from the `\(`, `\)`, `\[`, and `\]`, which, while it works, does make for some strange-looking page source.

Also, there are other locations that also need similar treatment (in webwork2).  In particular, 

https://github.com/openwebwork/webwork2/blob/05c982497e68d105829e190475ad73979df89c38/lib/WeBWorK/Utils/AttemptsTable.pm#L364-L366

and

https://github.com/openwebwork/webwork2/blob/05c982497e68d105829e190475ad73979df89c38/lib/WeBWorK/Utils/AttemptsTable.pm#L364-L366

Actually, it looks like there are other display modes that need similar treatment (e.g., the plainText and essay ones just above the linked lines).